src/app/post/index.tsx

```
@@ -0,0 +1,19 @@
+import { Error, Header, Loader, PostList } from '@/src/components';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)